npm/pkg/dataplane/dataplane_windows.go

```diff
@@ -334,9 +352,19 @@ func (dp *DataPlane) getLocalPodEndpoints() ([]*hcn.HostComputeEndpoint, error)
 	metrics.RecordListEndpointsLatency(timer)
 	if err != nil {
 		metrics.IncListEndpointsFailures()
-		return nil, npmerrors.SimpleErrorWrapper("failed to get local pod endpoints", err)
+		return nil, errors.Wrap(err, "failed to get local pod endpoints")
 	}
 
+	if dp.EnableNPMLite {
+		timer = metrics.StartNewTimer()
+		endpointsAttached, errL1vh := dp.ioShim.Hns.ListEndpointsQuery(dp.endpointQueryL1VH.query)
+		metrics.RecordListEndpointsLatency(timer)
+		if errL1vh != nil {
+			metrics.IncListEndpointsFailures()
+			return nil, errors.Wrap(errL1vh, "failed to get local pod endpoints in L1VH")
+		}
+		endpoints = append(endpoints, endpointsAttached...)
```

Comment:
Is there any risk if there are two of the same endpoint in this slice (two objects, same endpoint ID, different "state")? Not sure this would ever happen, but

Reply:
Added a function which takes out common endpoints and returns a unique list of endpoints